enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -63,6 +64,7 @@ It will not allow configuration of additional sources without further design con
 intended to provide a complex routing solution as one might achieve by using a custom collector configuration or a messaging solution (e.g. kafka).
 * It is not a goal for the tech-preview to support log forwarding outputs other then the ones identified for the goals. Admins can forward to their own fluentd via `forward` 
 and then configure that fluentd to forward to any number of specific logstore outputs
+* It is not a goal to provide secure storage for audit logs. If the deployer chooses to enable audit log forwarding, they need to make sure that the endpoint is compliant with governmental regulations and secure.
```

Comment:
"The OpenShift logging Elasticsearch does not comply with those regulations."

Reply:
Done! Thanks!